### DISCUSSION TITLE
Re: Shizuku won't uninstall!

### DISCUSSION THREAD

**Discussion by mr350** [samolego/Canta#75] [Q&A]
Shizuku won't uninstall!... https://github.com/RikkaApps/Shizuku/issues/526
Anyone know how to uninstall an App (Shizuku) that won't otherwise uninstall? 
Lenovo XiaoXin Pad 2022
Android 12
Device just restarts at any attempt to uninstall using device Settings, Apps, uninstall or ADB or Play store. Would rather not factory reset.

**Comment by mr350** [comment]
Could Canta uninstall Shizuku?

**Reply by samolego** [reply]
Might be able to, try.

**Comment by mr350** [comment]
I asked because Canta is powered by Shizuku...
**"Uninstall any app without root! Powered by Shizuku."
"How to Use... Install Shizuku & activate it."**
Shizuku is the App I'm trying to uninstall, would rather not install more software that then can't be uninstalled!

**Reply by samolego** [reply]
You can install Canta, try to uninstall Shizuku, then uninstall Canta normally.

**Reply by mr350** [reply]
I did try... pair wireless server, start server, authorize Canta app, try deleting... it does the same thing, just reboots the tablet! Shizuku doesn't uninstall!
At least Canta uninstalls! LOL!!!

**Reply by samolego** [reply]
Try reporting this to Shizuku directly.

**Reply by mr350** [reply]
I have! Look up! In my first post here!
https://github.com/RikkaApps/Shizuku/issues/526
